src/Logger.js

```diff
@@ -1,3 +1,5 @@
+import { performance } from 'perf_hooks';
```

Comment:
That's right, Node 6 doesn't have this. Let me know what you think about this change in general and I can update it so it works in Node 6 if you want this change.